wremnants/datasets/datagroups.py

```diff
@@ -53,17 +50,13 @@ def __init__(self, infile, combine=False, filter_datasets=True, mode=None, **kwa
             if mode not in mode_map.values():
                 raise ValueError(f"Unrecognized mode '{mode}.' Must be one of {set(mode_map.values())}")
             self.mode = mode
+        logger.info(f"Set mode to {self.mode}")
 
-        self.lumi = 1
```

Comment:
I see that this default is moved below, so I imagine it is fine, but I suggest running setupCombine without Data in the list of processes to ensure that if Data existed in the hdf5 file the luminosity is still correctly applied (and also that something reasonable happen when the Data histogram was not created at all).

Reply:
I tested running setupCombine with excluding data and also produced some plots without data and all looks as expected, the lumi is scaled to 1/fb